styles/markers.atom-text-editor.less

```diff
@@ -0,0 +1,8 @@
+:host(:host) .gutter .line-number {
+  &.line-red {
+    background-color: red;
+  }
+  &.line-yellow {
+    background-color: yellow;
```

Comment:
Same as above, except use `@background-color-warning` and `@text-color-warning` instead.